### PR TITLE
Bugfix confirm

### PR DESCRIPTION
**Pull request by AdamFigiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ A [WP-CLI](http://wp-cli.org/) command to rename WordPress' database prefix.
 
 ## Installing
 
-`wp package install iandunn/wp-cli-rename-db-prefix`
+`wp package install upsell/wp-cli-rename-db-prefix`
```
Comment (iandunn) [inline README.md:8]: What's the reason for this change?

File: wp-cli-rename-db-prefix.php [hunk 2]
```diff
@@ -28,292 +28,290 @@
...
-
-		foreach ( $sites as $site ) {
-			$update_query = $wpdb->prepare( "
+    public $old_prefix;
+    public $new_prefix;
+
+    public $is_dry_run = false;
+    public $is_prompt = true;
+    public $is_config_update = true;
+
```
Comment (iandunn) [inline wp-cli-rename-db-prefix.php:37]: You'll need to restore all the original spacing, etc, to remove all this diff noise. 

It's difficult to see what the changes are until that's done. The only modified lines in the diff should be the ones that you intend to change.

**Review by iandunn — CHANGES_REQUESTED** [review]

**Comment by iandunn** [comment]
I just remembered that there's already a PR for this: #16

File: wp-cli-rename-db-prefix.php (hunk 2 above)
Comment (szepeviktor) [reply wp-cli-rename-db-prefix.php:37]: GitHub has a cog wheel up there: https://github.com/iandunn/wp-cli-rename-db-prefix/pull/17/files?diff=unified&w=1
